application/src/main/java/org/thingsboard/server/install/ThingsboardInstallService.java

```diff
@@ -225,6 +225,10 @@ public void performInstall() {
                             log.info("Upgrading ThingsBoard from version 3.4.0 to 3.4.1 ...");
                             databaseEntitiesUpgradeService.upgradeDatabase("3.4.0");
                             dataUpdateService.updateData("3.4.0");
+                        case "3.4.1":
+                            log.info("Upgrading ThingsBoard from version 3.4.1 to 3.4.2 ...");
+                            databaseEntitiesUpgradeService.upgradeDatabase("3.4.1");
```

Comment:
Hi @YuriyLytvynchuk, please remove the following line since there are no DB schema changes in your PR.